srcpkgs/dotnet-sdk-bin/files/dotnet.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+export DOTNET_ROOT=/opt/dotnet
+export PATH="$PATH:$DOTNET_ROOT"
+# This seems to be needed to be able to work with LibreSSL
+export CLR_OPENSSL_VERSION_OVERRIDE=48
```

Comment:
Indeed, it would seem as if the LibreSSL updates might break this override is the version isn't properly maintainted. I can't think of a better work around for the time being. I'd love to hear your thoughts ~

Reply:
I'm not sure there is a proper non hacky one :/

Reply:
That does seem to be an issue working with these binaries over building from source. Given that this is the best one probably can do with these pre-built binaries, is it okay to resolve the converstaion?